junit-platform-commons/src/main/java/org/junit/platform/commons/support/conversion/ConversionSupport.java

```diff
@@ -30,17 +29,6 @@
 @API(status = EXPERIMENTAL, since = "1.11")
 public final class ConversionSupport {
```

Comment:
I realized now that there are no tests for `ConversionSupport` in `junit-platform-commons`, but `DefaultArgumentConverterTests` from `jupiter-tests` provide the corresponding coverage.

Is that intended or should I add something to `platform-tests`?

Reply:
No worries, I can take care of that!

Reply:
Raised #4305.